cv/qc-cv.obo

```diff
@@ -2137,3 +2136,12 @@ property_value: has_optional_column: QC:4000249 ! PCA Dimension 3
 property_value: has_optional_column: QC:4000250 ! PCA Dimension 4
 property_value: has_optional_column: QC:4000251 ! PCA Dimension 5
 
+
+[Term]
+id: QC:4000268
+name: Min. peaks for 50% of total spectra intensity
```

Comment:
Maybe use the common term "total ion current" instead of "spectra/spectrum intensity"?

Reply:
I was a bit quick with merging #139, so if we decide to change the term name, we shouldn't forget to modify the example mzML as well.

Reply:
> we shouldn't forget to modify the example mzML as well.

Certainly.
> common term "total ion current" instead of "spectra/spectrum intensity"?

I was afraid that this might not be taken in local context and cause confusion that way. I'm open to change it. What do @cbielow @julianu think?

Reply:
is this metric for a single spectrum (how would you reference the spectrum in a runQuality unless you are using a table?) or for a whole experiment (all spectra)?
This should be made clear in the description.

Depending on the above, TIC would make sense to me if its many spectra. If its only one spectrum, then I'm acutally not sure, since TIC always suggests a chromatogram (for me).

Reply:
also: `name: Min. *number of* peaks for ...`
?

Reply:
This is for a single spectrum. I think the term TIC is still correct in that case; the typical chromatogram plots are basically the TICs of all spectra combined.

I agree that it should be clearly specified that this is for a single spectrum only. Maybe this is even important enough to be a new categorization using a `relationship`. So rather than `single run`, as it is now, this would become `single spectrum` (with the corresponding new CV term). That seems more correct to me.

Reply:
This sounds like a good path to a solution, but still leaves
> (how would you reference the spectrum in a runQuality unless you are using a table?)

which is another good point. So initially we thought this one up to use in a mzML with mzQC example. There, each spectrum has a cvParamGroup to accommodate the metric. But of course we probably want to report these or other `single spectrum` metrics. 
How about a collective `single spectrum`s table metric instead of a extra table metric for each `single spectrum` metric? Call it `single spectrum metrics table`. And the table is a mandatory column of spectrum identifiers plus any `single spectrum` metric as optional column. However, it is quite conceivable to me that some single spectrum metrics are used for one particular subset of spectra and some for another subset. Since repeating a metric within a run/set is not allowed, we have to use the same metric for both subsets and following some blank spots in the table. If I remember correctly we haven't specified explicitly what to do with missing values table columns but I'd argue N/A for the case where no value was calculated and nan where the metric calculation went off the beaten path. What do you think?

Reply:
Section 9.3 discusses how to represent NaN. We don't mention N/A or null values yet.

A single spectrum table could be a solution. We'd have to think how to properly implement this though. For the optional columns, all of the possible values will need to be explicitly specified according to the current spec I think. And every time a new single spectrum metric is defined, this needs to be added as a optional column as well. That's going to become a mess pretty quickly. Can we somehow use inheritance or something in the CV to keep it maintainable?
The same kind of goes for the spectrum identifier: how do we represent multiple options (USI or nativeID, as mentioned in the spec)? One and only one should probably be used, so both columns shouldn't be specified as mandatory nor optional. Instead maybe two different single spectrum table terms in the CV, one when using USIs and one when using nativeIDs?

Reply:
All good points.

For the missing/uncomputable values:
nan seems to be unsuitable because it implies a number was expected, thus does not work well for strings etc.

How about: 
NULL for missing values 
NA for failed computation
?

This would be directly readable for R. Same for python?

Reply:
JSON has a built-in `null` value, so that can directly be used.

JSON and Python don't have a built-in NA value. Would just `null` work as well, without distinguishing between the source of the missingness?